deepmd/molecular_dynamics/job.pbs

```diff
@@ -15,25 +14,26 @@
 #    7. email about job abort, begin, end  (-m abe)                #
 #    8. email address to use               (-M)                    #
 #                                                                  #
-#  For more info visit https://docs.mat3ra.com/jobs-cli/overview/  #
+#  For more information visit https://docs.exabyte.io/cli/jobs     #
 # ---------------------------------------------------------------- #
 
-#PBS -N DEEPMD-LAMMPS-MD
+#PBS -N MD-LAMMPS
 #PBS -j oe
+##PBS -q comp-002-zone-a-production-20160630-cluster-007.exabyte.io
 #PBS -l nodes=1
-#PBS -l ppn=1
-#PBS -l walltime=00:00:10:00
-#PBS -q D
-#PBS -m abe
-#PBS -M info@exabyte.io
+#PBS -l ppn=4
+#PBS -l walltime=00:01:00:00
+#PBS -q OR
```

Comment:
For the queues - let's keep "D" as the calculation does fit there and the wait is shorter

Reply:
Some tasks (e.g., `model_training/job.pbs`) hit the memory limit of the debug queue.